.cron.yml

```diff
@@ -30,3 +30,9 @@ jobs:
           treeherder-symbol: screenshots-D
           target-tasks-method: screenshots
       when: [{weekday: 'Monday', hour: 10, minute: 0}]
+    - name: legacy-api-ui-tests
+      job:
+          type: decision-task
+          treeherder-symbol: legacy-api-ui
+          target-tasks-method: legacy_api_ui_tests
+      when: [{hour: 10, minute: 30}]
```

Comment:
Don't we have to set a day when this run? not sure how often we want this to run

Reply:
I think based on the other tasks defined in `.cron.yml` as example it will run daily if you omit an explicit date declare – which is fine for our purposes for now.